code/game/gamemodes/blob/overmind.dm

```diff
@@ -36,14 +36,17 @@
 	updateallghostimages()
 	..()
 
+/mob/camera/blob/Life()
+	if(!blob_core)	Destroy()

```

Comment:
1. calling destroy allows various list clean ups like mob_list, dead_mob_list, and living_mob_list to properly remove the overmind from them. This is handed right at line one of mob.dm

2. Tastes